### DISCUSSION TITLE
Including Cockpit Files are a Ubuntu package

### DISCUSSION THREAD

**Discussion by anapeksha** [cockpit-project/cockpit-files#712] [General]
Do we have any roadmap on including cockpit-files to be included as a package in Ubuntu 24.04?

**Comment by martinpitt** [comment]
It needs to be accepted into Debian (that's the big unknown) -- it's been in the [new queue](https://ftp-master.debian.org/new.html) for a month. Then synced into the current Ubuntu dev release (that's quick, I can do it), then reviewed by the Ubuntu backports team (which takes roughly a week).

**Reply by anapeksha** [reply]
Great! Glad to hear that it's in progress! Will be waiting for the release. Thanks for the reply
